cli_tools_cloudbuild.yaml

```diff
@@ -312,6 +312,64 @@ steps:
+  - --destination=southamerica-east1-docker.pkg.dev/$PROJECT_ID/wrappers/gce_ovf_export:$COMMIT_SHA
+  - --destination=europe-docker.pkg.dev/$PROJECT_ID/wrappers/gce_ovf_export:$_RELEASE
+  - --destination=europe-docker.pkg.dev/$PROJECT_ID/wrappers/gce_ovf_export:$COMMIT_SHA
+  - --destination=europe-north1-docker.pkg.dev/$PROJECT_ID/wrappers/gce_ovf_export:$_RELEASE
+  - --destination=europe-north1-docker.pkg.dev/$PROJECT_ID/wrappers/gce_ovf_export:$COMMIT_SHA
+  - --destination=europe-west1-docker.pkg.dev/$PROJECT_ID/wrappers/gce_ovf_export:$_RELEASE
+  - --destination=europe-west1-docker.pkg.dev/$PROJECT_ID/wrappers/gce_ovf_export:$COMMIT_SHA
+  - --destination=europe-west2-docker.pkg.dev/$PROJECT_ID/wrappers/gce_ovf_export:$_RELEASE
+  - --destination=europe-west2-docker.pkg.dev/$PROJECT_ID/wrappers/gce_ovf_export:$COMMIT_SHA
+  - --destination=europe-west3-docker.pkg.dev/$PROJECT_ID/wrappers/gce_ovf_export:$_RELEASE
+  - --destination=europe-west3-docker.pkg.dev/$PROJECT_ID/wrappers/gce_ovf_export:$COMMIT_SHA
+  - --destination=europe-west4-docker.pkg.dev/$PROJECT_ID/wrappers/gce_ovf_export:$_RELEASE
+  - --destination=europe-west4-docker.pkg.dev/$PROJECT_ID/wrappers/gce_ovf_export:$COMMIT_SHA
+  - --destination=europe-west6-docker.pkg.dev/$PROJECT_ID/wrappers/gce_ovf_export:$_RELEASE
+  - --destination=europe-west6-docker.pkg.dev/$PROJECT_ID/wrappers/gce_ovf_export:$COMMIT_SHA
+  - --destination=asia-docker.pkg.dev/$PROJECT_ID/wrappers/gce_ovf_export:$_RELEASE
+  - --destination=asia-docker.pkg.dev/$PROJECT_ID/wrappers/gce_ovf_export:$COMMIT_SHA
+  - --destination=asia-east1-docker.pkg.dev/$PROJECT_ID/wrappers/gce_ovf_export:$_RELEASE
+  - --destination=asia-east1-docker.pkg.dev/$PROJECT_ID/wrappers/gce_ovf_export:$COMMIT_SHA
+  - --destination=asia-east2-docker.pkg.dev/$PROJECT_ID/wrappers/gce_ovf_export:$_RELEASE
+  - --destination=asia-east2-docker.pkg.dev/$PROJECT_ID/wrappers/gce_ovf_export:$COMMIT_SHA
+  - --destination=asia-northeast1-docker.pkg.dev/$PROJECT_ID/wrappers/gce_ovf_export:$_RELEASE
+  - --destination=asia-northeast1-docker.pkg.dev/$PROJECT_ID/wrappers/gce_ovf_export:$COMMIT_SHA
+  - --destination=asia-northeast2-docker.pkg.dev/$PROJECT_ID/wrappers/gce_ovf_export:$_RELEASE
+  - --destination=asia-northeast2-docker.pkg.dev/$PROJECT_ID/wrappers/gce_ovf_export:$COMMIT_SHA
+  - --destination=asia-south1-docker.pkg.dev/$PROJECT_ID/wrappers/gce_ovf_export:$_RELEASE
+  - --destination=asia-south1-docker.pkg.dev/$PROJECT_ID/wrappers/gce_ovf_export:$COMMIT_SHA
+  - --destination=asia-southeast1-docker.pkg.dev/$PROJECT_ID/wrappers/gce_ovf_export:$_RELEASE
+  - --destination=asia-southeast1-docker.pkg.dev/$PROJECT_ID/wrappers/gce_ovf_export:$COMMIT_SHA
+  - --destination=australia-southeast1-docker.pkg.dev/$PROJECT_ID/wrappers/gce_ovf_export:$_RELEASE
+  - --destination=australia-southeast1-docker.pkg.dev/$PROJECT_ID/wrappers/gce_ovf_export:$COMMIT_SHA
+  - --context=/workspace
+  - --dockerfile=gce_ovf_export.Dockerfile
+  -
```

Comment:
Not sure about formatting for yaml files, but should there be anything following this dash or should it be removed?

Reply:
Good catch. I think it was auto added by the editor. Removed it now.